client/landing/stepper/declarative-flow/internals/steps-repository/components/flow-card/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Flex, FlexBlock, FlexItem, Card, CardBody, Icon } from '@wordpress/components';
+import { chevronRight } from '@wordpress/icons';
+import './style.scss';
+
+interface FlowCardProps {
+	icon: JSX.Element;
+	onClick: () => void;
+	text: string;
+	title: string;
+}
+
+const FlowCard = ( { icon, onClick, text, title }: FlowCardProps ) => {
+	return (
+		<Card className="flow-question" as="button" size="small" onClick={ onClick }>
+			<CardBody>
+				<Flex>
+					<FlexItem className="flow-question__icon">
+						<Icon icon={ icon } size={ 24 } />
```

Comment:
Maybe this doesn't need `Icon` component and can just have SVG in img tag instead. Would allow importing SVGs as files, not as JS, which reduces bundle size. But it's not very big deal.

We'll need to check screen-readers don't see this icon.

Reply:
Loading this as an icon component seems reasonable to me. I also just tested this screen with tabbing and screen reader voice over. It works pretty well - the icon isn't recognized for purposes of tabbing and navigation.